tests/IO/GeoJSONReaderTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Brick\Geo\Tests\IO;
+
+use Brick\Geo\GeometryCollection;
+use Brick\Geo\IO\GeoJSONReader;
+
+class GeoJSONReaderTest extends GeoJSONAbstractTest
+{
+    /**
+     * @dataProvider providerReadGeometry
+     *
+     * @param string $geojson    The GeoJSON to read.
+     * @param array  $coords     The expected Point coordinates.
+     * @param bool   $is3D       Whether the resulting Point has a Z coordinate.
+     * @param bool   $isMeasured Whether the resulting Point has a M coordinate.
```

Comment:
As far as I can tell, GeoJSON does not support the M (Measured) parameter, nor does GeoJSONReader which sets `$hasM = false`; plus all your tests only contain `false` for this parameter, which makes it useless.

As such, I would remove this parameter from the test methods, and remove the last parameter from every test provider.